lib/wibox/widget/calendar.lua

```diff
@@ -116,6 +121,11 @@ local properties = { "date", "font", "spacing", "week_numbers", "start_sunday",
 -- @param function Function to embed the widget depending on its flag
 -- @property fn_embed
 
+--- Allow cells to have flexible height
+--
+--@DOC_wibox_widget_calendar_flex_height_EXAMPLE@
```

Comment:
Does this example file even exist? Did you forget to commit it?

Reply:
This file has been added.